examples/gno.land/p/demo/fqname/fqname.gno

```diff
@@ -0,0 +1,71 @@
+// Package fqname provides utilities for handling fully qualified identifiers in
+// Gno. A fully qualified identifier typically includes a package path followed
```

Comment:
It might be good to include the proper definition of what does it mean in this context to be a "fully qualified" ID. Fully qualified for what?

Reply:
Isn't the full paragraph enough? I'm referring to the two lines after your comment.